ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.stories.js

```diff
@@ -18,15 +17,20 @@ export default {
       control: 'array',
       table: { category: 'Data' },
     },
+    symbol: {
+      control: 'text',
+      table: { category: 'Data' },
+    },
+    image: {
+      control: 'text',
+      table: { category: 'Data' },
+    },
   },
```

Comment:
I think we can add the eth logo here for all stories

```suggestion
  },
  args: {
    symbol: 'ETH',
    image: './images/eth_logo.svg',
  },
```